posts/2024/04/28/court-systems-research.mdx

```diff
@@ -0,0 +1,38 @@
+---
+
+<p className="lead">Over the last few weeks, we’ve been on a listening tour with the many admirable organizations who advance access to justice through technology, and want to improve court systems and processes. In line with our commitments to openness and transparency, we’re sharing what we have learned.</p>
+
+- **Everyone wants eFiling systems to better help the most common court user: self-represented litigants.** Although a few courts don’t offer eFiling, most have offered it for years. Many experts note that the strengths and weaknesses of the existing systems are well known, but we need a new approach to better meet the needs of the majority of court users: self-represented filers. 
+
+- **Siloed systems make it hard to streamline filing for non-lawyers.** Most existing eFiling systems present a few questions with jargon-filled dropdowns and a file upload field designed for lawyers. People without a lawyer guess what to enter, and often get it wrong. Many courts and assistance organizations try to help by providing step-by-step instructions, or a separate website that helps compile the necessary documents. These tools help, but the experience is still disjointed. And it turns out to be very hard to connect these systems together to create a single website to help non-lawyers. In sum, despite widespread eFiling, very few courts allow filers to answer questions about their problem and file a case in a single website. 
+
+- **Filer fees heavily subsidize filing systems, and that’s hard on self-represented litigants.** Courts don’t pay for most filing systems; the people courts serve do. Litigants have to “pay to play,” and that creates an additional barrier for many self-represented applicants. Applying for a fee waiver often forces busy litigants through another set of hoops. Many of our interviewees yearned for new business models for eFiling systems that don’t depend on user fees.
+
+- **Courts pay big to make small changes to systems that might help people.** Many courts and filer advocates want to change the text, flow or other details of their eFiling or case management system so it is clearer for everyone. But after the system is implemented, these changes are often very expensive. Sometimes, they even require new contracts. Even when a simple change could prevent filer mistakes, and save court timer, it often never happens.
+
+These are just a few of the findings in our report. To learn more, including a detailed overview of the way these systems fit together, read our full report.
+
+<div class="text-center"><a class="text-center whitespace-nowrap border border-transparent rounded-md shadow-sm text-base font-medium text-white text-sm no-underline bg-purple-800 hover:bg-purple-900 inline-flex px-6 py-4 text-lg" href="/pdf/Problems-Court-Filing-CMS-Perspectives-from-Advocates-and-Vendors.pdf">Read the report</a></div>
```

Comment:
There's a button widget you can use. Check out example.mdx.

Reply:
Done in [8102542](https://github.com/freelawproject/free.law/pull/180/commits/81025426fa6a3dfa9f70e31afe03779be19993ca)